doc/src/base/scopedvalues.md

```diff
@@ -0,0 +1,138 @@
+# Scoped Values
+
+Scoped values provide an implementation of dynamic scoping in Julia.
+Dynamic scope means that the state of the value is dependent on the execution path
+of the program. This means that for a scoped value you may observe
+multiple different values at the same time.
+
+!!! compat "Julia 1.11"
+    Scoped values were introduced in Julia 1.11. In Julia 1.8+ a compatible
+    implementation is available from the package ScopedValues.jl.
+
+In its simplest form you can create a [`ScopedValue`](@ref) with a
```

Comment:
I feel like it would be worth mentioning that `ScopedValue`s are global consts, which make them accessible in various program-depth call chains. It just might not be immediately obvious why you wouldn't create one of these at the start of a `main` function, for example.

Reply:
The fact that I use global constants as scoped values doesn't mean they need to be. You could absolutely create one of these at the start of a `main` function or store them inside a field of a struct.